fdbclient/NativeAPI.actor.cpp

```diff
@@ -1994,8 +1994,10 @@ ACTOR Future<Void> monitorNetworkBusyness() {
 			tracker.windowedTimer = now();
 		}
 
-		g_network->networkInfo.metrics.networkBusyness =
-		    std::min(elapsed, tracker.duration) / elapsed; // average duration spent doing "work"
+		double busyFraction = std::min(elapsed, tracker.duration) / elapsed;
+		double burstiness = std::min(1.0, std::max(0.0, tracker.maxDuration - 0.1) / 0.4);
```

Comment:
This is currently just an example of something that accounts for bursty behavior, but for now it's not set in stone. It gradually ramps up the busyness score as the longest non-idle period increases from 100ms to 500ms.